desktop/src/main/java/bisq/desktop/main/overlays/windows/DisputeSummaryWindow.java

```diff
@@ -811,6 +812,51 @@ public void onFailure(TxBroadcastException exception) {
         }
     }
 
+    private void doCloseAfterTxsVerified(Button closeTicketButton) {
+        var disputeManager = getDisputeManager(dispute);
+        // Only RefundAgent need to verify transactions to ensure payout is safe
+        if (disputeManager instanceof RefundManager && Config.baseCurrencyNetwork().isMainnet()) {
+            RefundManager refundManager = (RefundManager) disputeManager;
+            Contract contract = dispute.getContract();
+            String makerFeeTxId = contract.getOfferPayload().getOfferFeePaymentTxId();
+            String takerFeeTxId = contract.getTakerFeeTxID();
+            String depositTxId = dispute.getDepositTxId();
+            String delayedPayoutTxId = dispute.getDelayedPayoutTxId();
+            Popup requestingTxsPopup = new Popup().feedback(Res.get("disputeSummaryWindow.requestingTxs"));
+            requestingTxsPopup.show();
+            refundManager.requestBlockchainTransactions(makerFeeTxId,
+                    takerFeeTxId,
+                    depositTxId,
+                    delayedPayoutTxId
+            ).whenComplete((txList, throwable) -> {
+                UserThread.execute(() -> {
+                    if (throwable == null) {
+                        try {
+                            requestingTxsPopup.hide();
```

Comment:
`requestingTxsPopup.hide();` should be moved up 2 lines, before the if/else section so that it is also invoked for the failure scenario.  Otherwise the popup prevents the next one (the requestTransactionsError) from being shown.

Reply:
Ah good point.